templates/privacyidea.xhtml

```diff
@@ -0,0 +1,98 @@
+<!DOCTYPE composition
+	PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
+<ui:composition xmlns="http://www.w3.org/1999/xhtml" xmlns:f="http://xmlns.jcp.org/jsf/core"
+	xmlns:ui="http://xmlns.jcp.org/jsf/facelets" xmlns:h="http://xmlns.jcp.org/jsf/html"
+	template="/WEB-INF/incl/layout/login-template.xhtml">
+
+	<f:metadata>
+		<f:viewAction action="#{authenticator.prepareAuthenticationForStep}" />
+		<f:viewParam name="login_hint" value="#{loginAction.loginHint}" />
+	</f:metadata>
+	<ui:define name="pageTitle">
+		<h:outputText value="privacyIDEA - oxAuth" />
+	</ui:define>
+	<ui:define name="body">
+		<div class="container">
+			<h:panelGroup rendered="true">
+				<div class="login_bx_1"
+					style="border-radius: 10px; margin-top: 0px; background: white; border: 1px solid #008b8b;">
+					<div class="row">
+						<h:messages class="text-center"
+							style="color:#4ca4ec;margin:5px;margin-left:20px; font-size:1vw;" infoClass="text-center"
+							errorClass="text-center" />
+					</div>
+					<h:form id="loginForm" style="padding:30px;" prependId="false">
+						<div class="form-group row"></div>
+						<div class="col-sm-offset-2 offset-md-2 col-sm-8 col-md-8">
+							<p>
+								<h:outputText value="#{identity.getWorkingParameter('transaction_message')}" />
+							</p>
+						</div>
+						<div class="form-group row">
+							<div class="col-sm-offset-2 offset-md-2 col-sm-8 col-md-8">
+								<h:inputSecret placeholder="OTP" colMd="10" id="otp" name="otp" labelColMd="2"
```

Comment:
Hm, where is actually the username, or is this only for chal-resp?

Reply:
The username is persisted in the session from step 1, where the default gluu login page is used.

Reply:
I was assuming this. Thanks for clarification.